actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -38,6 +38,7 @@ def with_subscriptions_connection(&block) # :nodoc:
           # will perform the necessary cleanup tasks
           ActiveRecord::Base.connection_pool.remove(conn)
         end
+        ar_conn.connect!
```

Comment:
This feels surprising... `raw_connection` should probably take care of it?

Reply:
Possibly, I was trying various stuff for fixing Action Cable, this probably isn't necessary, see: https://github.com/rails/rails/pull/49414